ShowBuildMenu.bat

```diff
@@ -182,7 +182,7 @@ SET NUNITPLATFORM=
 goto test-run
 
 :test-run
-start "nunit3-console" cmd /K %NUNIT% %NUNITPLATFORM% --agents=1 --process=separate NHibernate.nunit
+start "nunit3-console" cmd /K %NUNIT% %NUNITPLATFORM% --agents=1 NHibernate.nunit
```

Comment:
Removing that option default to multiple, which by default runs assemblies in parallel. But along with the `agents=1`, they will still be run sequentially, so it is not an issue to remove the `process` option.